sql/firefox_nondesktop_exact_mau28_by_product_by_dimensions_v1.sql

```diff
@@ -39,7 +43,7 @@ WHERE
   AND os IN ('Android', 'iOS')
   AND normalized_channel = 'release'
   -- 2017-01-01 is the first populated day of telemetry_core_parquet, so start 28 days later.
-  AND @submission_date >= '2017-01-28'
+  AND @submission_date >= DATE('2017-01-28')
```

Comment:
I like `DATE '2017-01-28'` better, but for no reason. Do you have a strong preference? Should we standardize on one?

Reply:
I didn't fully absorb this comment earlier. I wasn't even aware of the `DATE '2019-01-01'` syntax. You're already using that consistently, so I'm fine to keep with that as I make changes in the future.